Tema1/blog.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE HTML>
+<html lang = "en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Blog</title>
+<link rel="shortcut icon" type = "img/png" href="img/turn.png">
+<link rel = "Stylesheet" href = "styles.css">
+
+</head>
+
+<body style="background: url('https://images.unsplash.com/photo-1516053256215-94022213b13c?q=80&w=1974&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D')">
+    <head>
+        <p> <h2>Acesta este locul in care imi place sa impartasesc articolele preferate.
+        </h2></p>
+        <p>Sunt pasionata de psihologia copiilor, retete pentru intreaga familie si seriale.</p>
+    </head>
```

Comment:
tag-ul `<head>` nu poate fi folosit in cadrul tag-ului `<body>`, aici poate ar fi mers mai bine tag-ul de `<header>` ce ar reprezenta antetul paginii de blog
```suggestion
    <header>
        <p> <h2>Acesta este locul in care imi place sa impartasesc articolele preferate.
        </h2></p>
        <p>Sunt pasionata de psihologia copiilor, retete pentru intreaga familie si seriale.</p>
    </header>
```